### PR TITLE
Minor code fixes & renamings in host.i.dfy

### PR DESCRIPTION
**Pull request by teoparvanov**
With this PR I suggest to:
* rename some important predicates for better code readability
* add what looks to me like missing "well-formed" validations (unless they are redundant?)

### CODE REVIEW

**Comment by evdzhurov** [comment]
The renaming makes sense to me, but ultimately Hristo must say because from what we've discussed there are a lot of parts of the code with temporary names and the intention should be clarified.

**Comment by HristoStaykov** [comment]
> The renaming makes sense to me, but ultimately @HristoStaykov must say because from what we've discussed there are a lot of parts of the code with temporary names and the intention should be clarified.
> 
> As for the WF checks since they are included in the sub-predicates that you renamed you would be duplicating them I think. The expressions prior to RecvPre/PrepareEnabled don't use 'v' and 'c'

There was one temporary name in the proof section, which is a WIP. We do try to keep them as clear as possible prior to finalizing the first proof, when they will once again be revised.

File: host.i.dfy [hunk 1]
```diff
@@ -124,17 +124,18 @@ module Host {
   // predicate we need to reflect in our next state that we have received the PrePrepare message.
   predicate RecvPrePrepare(c:Constants, v:Variables, v':Variables, msgOps:Network.MessageOps<Message>)
   {
+    && v.WF(c)
     && msgOps.recv.Some?
     && msgOps.send.None?
     && var msg := msgOps.recv.value;
-    && RecvPrePrepareEnabled(c, v, msg)
+    && IsValidPrePrepare(c, v, msg)
```
Comment (teoparvanov) [inline host.i.dfy:131]: IsValidPrePrepareToAccept?

File: host.i.dfy [hunk 2]
```diff
@@ -124,17 +124,18 @@ module Host {
...
     && v' == v.(workingWindow := 
                 v.workingWindow.(prePreparesRcvd := 
                                  v.workingWindow.prePreparesRcvd[msg.seqID := Some(msg)]))
   }
 
   // For clarity here we have extracted all preconditions that must hold for a Replica to accept a Prepare
-  predicate RecvPrepareEnabled(c:Constants, v:Variables, p:Message)
+  predicate IsValidPrepare(c:Constants, v:Variables, p:Message)
```
Comment (teoparvanov) [inline host.i.dfy:138]: IsValidPrepareToAccept?

**Review by HristoStaykov — APPROVED** [review]
LGTM, thanks @teoparvanov !

**Pull request merged**